src/main/drivers/accgyro_fake.c

```diff
@@ -23,23 +23,6 @@
 #if defined(SIMULATOR_BUILD) && defined(SIMULATOR_MULTITHREAD)
 #include <stdio.h>
 #include <pthread.h>
-#define printf printf
-#define sprintf sprintf
```

Comment:
We need these two define, otherwise it won't print to console because `common/printf.h` had define it too.